model/hardware_schema.json

```diff
@@ -7,15 +7,19 @@
     "properties": {
       "componentName": {
         "type": "string",
-        "description": "the type of this subsystem part of your infrastructure, example: cpu, gpu, hdd, sdd..."
+        "description": "the name of this subsystem part of your infrastructure, example returned by codecarbon: 1 x NVIDIA GeForce GTX 1080 Ti"
+      },
+      "componentType": {
+        "type": "string",
+        "description": "the type of this subsystem part of your infrastructure, example: cpu, gpu, ram, hdd, sdd..."
       },
       "nbComponent": {
         "type": "integer",
         "description": "the number of items of this component in your infrastructure"
       },
       "memorySize": {
         "type":"integer",
-        "description": "the size of the memory of the component in Gbytes"
+        "description": "the size of the memory of the component in Gbytes, useful to detail the memory associated to ONE of your gpus for example (if we want the total memory, we will multiply the memorySize by nbComponent)."
```

Comment:
Ici j'aurais ajouté : if the component is CPU do not put here the RAM size, create another component for the RAM this field is for the embeded memory of a component.